### PR TITLE
Define a team/katacoda label for k/website

### PR DESCRIPTION
**Pull request by sftim**
[Katacoda](https://www.katacoda.com/) provide hosted Kubernetes learning environments for https://kubernetes.io/docs/tutorials/ - this label will filtering on those specific issues.

We already have the label, but it's manually-managed and Prow doesn't recognize it.

### CODE REVIEW

File: label_sync/labels.yaml
Comment (spiffxp) [inline label_sync/labels.yaml:1499]: Note AFAIK you also need to update prow config to allow people to /label team/katacoda

Comment (spiffxp) [reply label_sync/labels.yaml:1499]: It's likely plugins.yaml

**Review by spiffxp — APPROVED** [review]
Need to fix tests and then I'll lgtm for real
/hold
Remove when updated

**Comment by sftim** [comment]
Might take a while before I can get this right, unless there's a container made that has the relevant tools ready to use.

**Comment by fejta** [comment]
/uncc
LGTM

**Comment by sftim** [comment]
/hold cancel
as generated Markdown is now fixed

**Pull request merged**
